### PR TITLE
Remove the client blocklist options

### LINKED ISSUE #3914: Remove the client blocklist options
- [x] Remove `blacklist_ip_addresses`
- [x] Remove `blacklist_user_agents`

see #3909

### PR DESCRIPTION
**Pull request by TimWolla**
Resolves #3914

### CODE REVIEW

File: com.woltlab.wcf/option.xml [hunk 1]
```diff
@@ -1701,5 +1693,8 @@ DESC:wcf.global.sortOrder.descending</selectoptions>
 	</import>
 	<delete>
 		<option name="module_master_password"/>
+		<option name="blacklist_ip_addresses"/>
+		<option name="blacklist_user_agents"/>
+		<!-- <optioncategory name="security.blacklist.custom"/> -->
```
Comment (TimWolla) [inline com.woltlab.wcf/option.xml:1698]: Not sure about this one. It might be in use by plugins.

Comment (dtdesign) [reply com.woltlab.wcf/option.xml:1698]: An empty category doesn't hurt, we can deprecate it, but keep it for the 5.x series around.

**Review by dtdesign — APPROVED** [review]

File: com.woltlab.wcf/option.xml (hunk 1 above)
Comment (Gravatronics) [reply com.woltlab.wcf/option.xml:1698]: If it is not removed, I would this commented.out line.

File: com.woltlab.wcf/option.xml [hunk 2]
```diff
@@ -714,14 +714,6 @@ block:wcf.acp.option.blacklist_sfs_action.block]]></selectoptions>
...
 			<!-- security.blacklist.custom -->
-			<option name="blacklist_ip_addresses">
-				<categoryname>security.blacklist.custom</categoryname>
-				<optiontype>textarea</optiontype>
-			</option>
-			<option name="blacklist_user_agents">
-				<categoryname>security.blacklist.custom</categoryname>
-				<optiontype>textarea</optiontype>
-			</option>
 			<!-- /security.blacklist.custom -->
```
Comment (Gravatronics) [inline com.woltlab.wcf/option.xml:717]: Also remove the `security.blacklist.custom` comments.

**Review by Gravatronics — CHANGES_REQUESTED** [review]
Everything else LGTM.

**Pull request merged**